main.js

```diff
@@ -305,6 +316,8 @@ app.get('/profile', async (req, res) => {
 	profile = changeLoginButton(profile, req);
 
 	profile = await loadHTMLComponent(profile, "#Base-Container", "div", "./templates/profile.html");
+	profile = await loadHTMLComponent(profile, "#kit-templates", "div", "./templates/kit.html");
```

Comment:
I've been thinking about it: a better design for the component-loading thing might be to have the HTML files themselves specify what elements should get templated content put into them. For example:

```html
<html>
<body>
<div id="template-container" data-template-target="./templates/profile.html">
</div>
</body>
</html>
```

Gets transformed by some magic function like `loadAllTemplates(JSDOM): JSDOM` into:

```html
<html>
<body>
<div id="template-container">
<div>I used to be in a template! Yayy!</div>
</div>
</body>
</html>
```

Obviously some manual code might be required for more complicated things, but maybe you see the point.

Reply:
(And, to be clear, perhaps this is a thing to do in the next sprint, not a critical issue.)